pages/login.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FC } from 'react';
+import Page from '../components/Page';
+import { Form, Input, Button, Checkbox } from 'antd';
+import { MailOutlined, LockOutlined } from '@ant-design/icons';
+import axios from '../axios';
+
+const Login: FC = () => {
+
+    const onFinish = values => {
+        const { email, password } = values;
```

Comment:
At the moment, the backend accepts username only, update email with username for now.
Once this is integrated with the backend, we can change it accept both.